app/views/layouts/apipie/apipie.html.erb

```diff
@@ -1,26 +1,24 @@
 <!DOCTYPE html>
 <html>
-<head>
-  <title>API documentation</title>
-  <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
-  <%= Apipie.include_stylesheets %>
-  <link type='text/css' rel='stylesheet' href='<%= Apipie.full_url("stylesheets/application.css")%>'/>
-  <!-- IE6-8 support of HTML5 elements -->
-  <!--[if lt IE 9]>
-    <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
-  <![endif]-->
-</head>
-<body>
-  <div class="container">
-    <div class="row">
-      <div id='container'>
+  <head>
+    <title>API documentation</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"/>
+    <%= Apipie.include_stylesheets %>
+    <link type='text/css' rel='stylesheet' href='<%= Apipie.full_url("stylesheets/application.css")%>'/>
+    <!-- IE6-8 support of HTML5 elements -->
+    <!--[if lt IE 9]>
+      <script src="http://html5shim.googlecode.com/svn/trunk/html5.js"></script>
+    <![endif]-->
+  </head>
+  <body>
+    <div class="container-fluid">
+      <div class="row-fluid">
```

Comment:
Oh never mind, the bootstrap included is version 2, for which this syntax is correct.